sycl/test-e2e/Basic/info.cpp

```diff
@@ -327,6 +327,9 @@ int main() {
   print_info<info::device::driver_version, std::string>(dev, "Driver version");
   print_info<info::device::profile, std::string>(dev, "Profile");
   print_info<info::device::version, std::string>(dev, "Version");
+  if (dev.get_platform().get_backend() != backend::ext_oneapi_hip)
```

Comment:
why don't you handle this query in the HIP backend instead?

Reply:
+1 here, better to add at least a dummy implementation returning an empty string for HIP + FIXME in the code